src/libraries/System.Private.CoreLib/gen/EventSourceGenerator.Emitter.cs

```diff
@@ -12,97 +12,86 @@ namespace Generators
 {
     public partial class EventSourceGenerator
     {
-        private sealed class Emitter
-        {
-            private readonly StringBuilder _builder = new StringBuilder(1024);
-            private readonly GeneratorExecutionContext _context;
-
-            public Emitter(GeneratorExecutionContext context) => _context = context;
-
-            public void Emit(EventSourceClass[] eventSources, CancellationToken cancellationToken)
-            {
-                foreach (EventSourceClass? ec in eventSources)
-                {
-                    if (cancellationToken.IsCancellationRequested)
-                    {
-                        // stop any additional work
-                        break;
-                    }
+        /// <summary>Code for a [GeneratedCode] attribute to put on the top-level generated members.</summary>
+        private static readonly string s_generatedCodeAttribute = $"[global::System.CodeDom.Compiler.GeneratedCodeAttribute(\"{typeof(EventSourceGenerator).Assembly.GetName().Name}\", \"{typeof(EventSourceGenerator).Assembly.GetName().Version}\")]";
 
-                    _builder.AppendLine("using System;");
-                    GenType(ec);
+        private static void EmitSourceFile(SourceProductionContext spc, EventSourceClass ec)
```

Comment:
💡 Convention for analyzers/code fixes/refactorings/source generators is to use `context` as the parameter name for all input contexts.

```suggestion
        private static void EmitSourceFile(SourceProductionContext context, EventSourceClass ec)
```

Reply:
Done.